app/models/feedback.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Feedback
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  RATINGS = {
+    5 => 'Very satisfied',
+    4 => 'Satisfied',
+    3 => 'Neither satisfied nor dissatisfied',
+    2 => 'Dissatisfied',
+    1 => 'Very dissatisfied'
+  }.freeze
+
+  attr_accessor :comment, :rating, :email
+  validates :comment, presence: true
```

Comment:
Has a decision been made to enforce a comment? CCCD does not seem to do this for example

Reply:
👍 I have made comment optional as per James Costons comment that fields such as these are usually optional.